backend/handlers/handlers.go

```diff
@@ -1079,17 +1070,24 @@ func (handlers *Handlers) getExchangesByRegion(r *http.Request) interface{} {
 
 func (handlers *Handlers) getBitsuranceLookup(r *http.Request) interface{} {
```

Comment:
generally state-changing HTTP calls should be done as POST requests for clarity/consistency.

i'd switch it to be a POST call instead.